sotoki/sotoki.py

```diff
@@ -1677,11 +1677,12 @@ def run():
     if tag_depth != -1 and tag_depth <= 0:
         sys.exit("--tag-depth should be a positive integer")
     domain = arguments["<domain>"]
-    if re.match("^https?://", domain):
-        url = domain
-        domain = re.sub("^https?://", "", domain).split("/")[0]
-    else:
-        url = "http://" + domain
+    if re.match("^https?://",domain):
```

Comment:
please respect the code formatting ; ie. space after a comma.

Reply:
I will surely be keeping this in mind from the next time.